### PR TITLE
New command GCP Compute Engine

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/22529

## Description
Added new command **gcp-compute-project-info-add-metadata** - Add or update project-wide metadata.
Additional changes: fixed various bugs while creating the tests and documentation.

## Minimum version of Demisto
- [x] 4.5.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by DeanArbel** [comment]
@kirbles19 the following files require technical writer review:
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/0dc0e7b5cd8d8c2e4bc75d6de50e527db1e7e0f8/Packs/Legacy/Integrations/GoogleCloudCompute/CHANGELOG.md)
- [ ] [yml file](https://github.com/demisto/content/blob/0dc0e7b5cd8d8c2e4bc75d6de50e527db1e7e0f8/Packs/Legacy/Integrations/GoogleCloudCompute/GoogleCloudCompute.yml)
- [ ] [README](https://github.com/demisto/content/blob/0dc0e7b5cd8d8c2e4bc75d6de50e527db1e7e0f8/Packs/Legacy/Integrations/GoogleCloudCompute/README.md)

**Comment by yuvalbenshalom** [comment]
@DeanArbel check out the build

**Comment by kirbles19** [comment]
@DeanArbel Done. the YML looked good, RN comments made.

File: Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.py [hunk 1]
```diff
@@ -4331,264 +4321,306 @@ def set_snapshot_labels(args):
...
 
+def add_project_info_metadata(metadata):
+    """
+    Add or update project wide metadata.
+    :param metadata:  Each metadata entry is a key/value pair separated by ';' like so: key=abc,value=123;key=abc,value=123
+    """
+    project = SERVICE_ACT_PROJECT_ID
+    project_instance = get_compute().projects().get(project=project).execute()
+    fingerprint = project_instance.get('tags', {}).get('fingerprint')
+    items = parse_labels(metadata)
```
Comment (yuvalbenshalom) [inline Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.py:4332]: how dose the error look to the user if the metadata was in a bad format?
I dont see a special handling. I think the user should get an error that shows the actual content and suggest the correct format.

Comment (DeanArbel) [reply Packs/GoogleCloudCompute/Integrations/GoogleCloudCompute/GoogleCloudCompute.py]: @yuvalbenshalom 
image

**Pull request merged**